### PR TITLE
Increase nav timeout , added new logging.

### PR DESCRIPTION
**Pull request by izelaciman**

### CODE REVIEW

File: config/custom-environment-variables.json [hunk 1]
```diff
@@ -50,5 +50,7 @@
       }
     }
   },
-  "apiCallTimeout" : "API_CALL_TIMEOUT"
+  "apiCallTimeout" : "API_CALL_TIMEOUT",
+  "navigationTimeout(" : "NAVIGATION_TIMEOUT"
```
Comment (t0nin0s) [inline config/custom-environment-variables.json]: typo on navigationTimeout key?

File: config/default.json [hunk 2]
```diff
@@ -52,5 +52,6 @@
       }
     }
   },
-  "apiCallTimeout" : 40000
+  "apiCallTimeout" : 40000,
+  "navigationTimeout" : 180000
```
Comment (chrisgrimble) [inline config/default.json]: Is this not now the same as the mocha timeout? If so won't mocha timeout before this as we make multiple page requests per test?

Comment (izelaciman) [reply config/default.json]: I mean theoretically yes :)  trying to figure out if this is the actual cause 180 sec is already too high for both anyway will decrease it to 140 sec just too see if this is the case...

Comment (izelaciman) [reply config/default.json]: or we can also increase mocha time out even more but then I believe pipeline has a time limit of 40 min :)

**Pull request merged**